src/renderer/components/ft-select/ft-select.vue

```diff
@@ -19,9 +19,9 @@
     />
     <span class="select-highlight" />
     <span class="select-bar" />
-    <label class="select-label">
+    <p class="select-label">
```

Comment:
Should clicking on one of the labels focus the select?

Reply:
Can you clarify the question?

Reply:
I'm just curious why you switched to paragraph tags instead of adding a "for" attribute to the label. 

I don't think the paragraphs are binding properly with the form elements (when you click on a label it normally focuses the associated input element & this doesn't happen with the paragraphs).

This link describes the importance of labels for accessibility:
https://developer.mozilla.org/en-US/docs/Learn/Forms/How_to_structure_a_web_form#the_<label>_element

This link has good alternatives to label (ex. Adding `v-bind:aria-labelby="placeholder"` to the select elements ):
https://www.w3.org/WAI/tutorials/forms/labels/

Reply:
You're completely correct, fixed now. I thought that for some reason our implementation did not support this behavior. I tried again after you posted this and fiddled around for an hour trying to figure out why it was not working. It turns out that the label was given the `pointer-events: none` style in `ft-select.css`.